js/client-teambuilder.js

```diff
@@ -1245,6 +1254,39 @@
 			this.save();
 		},
 
+		setNotesBlur: function (e) {
+			var textarea = $(e.currentTarget);
+			if (!$.trim(textarea.val())) textarea.parent().hide();
+			if (textarea.val().indexOf('|') >= 0 || textarea.val().indexOf(']') >= 0 || textarea.val().indexOf('\t') >= 0) {
+				app.addPopupMessage("Unfortunately, set notes can't contain tabs or the characters | and ], since they're used for storing teams.");
```

Comment:
Right, the branch is still called team-notes because that's what I initially tried, I switched because set notes were way more convenient to implement regarding exporting/importing.

> doing .replace(/\t(.*)\t/g, '') before passing it to the packed team decoder.

I'm not sure I understand. Do you mean doing that before passing the packed team to the server with `/utm`? That would indeed be a good solution for that case. But the packed team needs to be decoded client-side for the teambuilder as well, and just erasing the set notes is not a good idea there.